templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,35 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List <a style="margin-left: 20px" href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
```

Comment:
The URL name 'manufacturer-create' is correctly using a hyphen to separate words, which aligns with the checklist requirement.